src/main/kotlin/uk/gov/dluhc/notificationsapi/service/SendNotificationService.kt

```diff
@@ -42,20 +43,29 @@ class SendNotificationService(
             val sentNotification = sendNotificationForChannel(requestDto, personalisationMap, notificationId, sentAt)
             saveSentMessageAndCreateAuditOrLogError(sentNotification)
             if (shouldSendApplicationStatisticsUpdate(requestDto)) {
-                statisticsUpdateService.triggerVoterCardStatisticsUpdate(requestDto.sourceReference)
+                statisticsUpdateService.triggerStatisticsUpdate(requestDto.sourceReference, requestDto.sourceType)
             }
         } catch (ex: GovNotifyNonRetryableException) {
             logger.warn("Non-retryable error returned from the Notify service: ${ex.message}")
         }
     }
 
-    private fun shouldSendApplicationStatisticsUpdate(requestDto: SendNotificationRequestDto): Boolean =
+    private fun shouldSendApplicationStatisticsUpdateForNotificationType(requestDto: SendNotificationRequestDto): Boolean =
         when (requestDto.notificationType) {
             NotificationType.ID_DOCUMENT_REQUIRED -> true
             NotificationType.ID_DOCUMENT_RESUBMISSION -> true
             NotificationType.ID_DOCUMENT_RESUBMISSION_WITH_REASONS -> true
             NotificationType.PHOTO_RESUBMISSION -> true
             NotificationType.PHOTO_RESUBMISSION_WITH_REASONS -> true
+            NotificationType.REQUESTED_SIGNATURE -> true
+            else -> false
+        }
+
+    private fun shouldSendApplicationStatisticsUpdate(requestDto: SendNotificationRequestDto): Boolean =
```

Comment:
I couldn't see any logic before that would prevent stats being added to the votercard queue for all sourceTypes? 
Unsure if I've missed something or if that was happening, but perhaps it was fine because the application ID wouldn't match a VC app ID so it wouldn't ultimately be added to the db

Reply:
Yep, I think that's a good description of what was happening.